cpp/server/ct-server.cc

```diff
@@ -222,10 +231,30 @@ int main(int argc, char* argv[]) {
   // For now, run with a dedicated thread pool as the executor for our
   // consistent store to avoid the possibility of DoS through thread starvation
   // via HTTP.
-  ThreadPool etcd_pool(2);
-  EtcdConsistentStore<LoggedCertificate> consistent_store(&etcd_pool,
+  ThreadPool internal_pool(4);
+  EtcdConsistentStore<LoggedCertificate> consistent_store(&internal_pool,
                                                           &etcd_client,
                                                           "/root", node_id);
+  // Put a sensible single-node config into FakeEtcd. For a real clustered log
+  // we'd expect a ClusterConfig already to be present within etcd as part of
+  // the provisioning of the log.
+  {
+    ct::ClusterConfig config;
+    config.set_minimum_serving_nodes(1);
+    config.set_minimum_serving_fraction(1);
+    LOG(INFO) << "Setting default single-node ClusterConfig:\n"
+              << config.DebugString();
+    consistent_store.SetClusterConfig(config);
```

Comment:
For the record, we discussed this a bit in person, and we might have to revisit later, but this is okay for now.

A big missing feature (which will be coming very soon) is for the cluster state controller not to operate without a config file (at the moment, it will run for a bit with the protobuf-supplied defaults of zero!).